src/sentry/utils/committers.py

```diff
@@ -199,7 +199,11 @@ def get_event_file_committers(project, event, frame_limit=25):
     if event.platform == 'java':
         for frame in frames:
             if '/' not in frame.get('filename') and frame.get('module'):
-                frame['filename'] = frame['module'].replace('.', '/') + '/' + frame['filename']
+                # Replace the last module segment with the filename, as the
+                # terminal element in a module path is the class
+                module = frame['module'].split('.')
+                module[-1] = frame['filename']
```

Comment:
Let me make certain I know why this is being done as I don't think it's self evident. In the example: 
```json
{
   "module": "io.sentry.example.Application",
   "abs_path": "Application.java",
   "filename": "Application.java",
}
```
We change it so that the output here will be:
```python
frame['filename'] = 'io/sentry/example/Application.java'
```
Are we always guaranteed that the `frame['filename']` is just the name of the file? So that we cannot get:
```python
frame['filename'] = 'io/sentry/example/io.sentry.example.Application.java'
```
Or some partial part of the above?

Reply:
Yep you're spot on with what the code is doing. The filename property comes from Java's [StackTraceElement.getFileName](https://docs.oracle.com/javase/7/docs/api/java/lang/StackTraceElement.html#getFileName()) which is documented as returning the file name or file path. If we get a path containing at least one `/` we don't mangle the module name into a path.